DOCS-#2971: Document `modin.distributed.dataframe.pandas` (#3019)
modin-project/modin: docs/flow/modin/distributed/dataframe/pandas.rst

Pandas Partition API
====================

> This page contains description of API to operate on Modin DataFrame partitions that are pandas DataFrame(s).

Reviewer: ```suggestion
This page contains a description of the API to extract partitions from and build Modin Dataframes.
```
